README.md

```diff
@@ -76,9 +80,11 @@ while running:
             elif msg.error().code() != KafkaError._PARTITION_EOF:
                 print(msg.error())
                 running = False
-    except SerializerError:
-        print("Unable to decode the message")
+    except SerializerError as e:
+        exc_type, exc_value, exc_traceback = sys.exc_info()
```

Comment:
This is nice, but maybe a bit excessive for an example, I just ment to print the exception string, something like:
` print("Message deserialization failed for %s: %s" % (msg, e))`